### PR TITLE
new org policies

### PR DESCRIPTION
**Pull request by mauro-cit**

### CODE REVIEW

File: 1-org/envs/shared/org_policy.tf [hunk 1]
```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
...
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/iam.disableServiceAccountKeyUpload"
+}
+
+data "google_organization" "org" {
+  organization = var.org_id
```
Comment (daniel-cit) [inline 1-org/envs/shared/org_policy.tf]: This should not use the `var.org_id`, 
the user must provide a list of domains, like it is done in the domain restricted sharing. 
https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/modules/domain_restricted_sharing/main.tf#L17-L20

and it must be a different list of the list used for domain restricted sharing

File: 1-org/envs/shared/org_policy.tf [hunk 2]
```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
...
+module "restrict_procotol_fowarding" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = ["internal"]
+  enforce         = "true"
+  constraint      = "constraints/compute.restrictProtocolForwardingCreationForTypes"
```
Comment (daniel-cit) [inline 1-org/envs/shared/org_policy.tf]: each new org policy should be placed in the correct section base in its type: `compute` .`iam` or `essentialcontacts`.
`essentialcontacts` can be grouped with the domain restrict sharing

File: 1-org/envs/shared/org_policy.tf [hunk 3]
```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
   parent = "organizations/${var.org_id}"
   title  = "default policy"
 }
+
+
+module "restrict_procotol_fowarding" {
```
Comment (daniel-cit) [inline 1-org/envs/shared/org_policy.tf]: it is necessary to set `allow_list_length` to 1

- https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/variables.tf#L80-L83
- https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/list_constraints.tf#L165
- https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/list_constraints.tf#L181

File: 1-org/envs/shared/org_policy.tf [hunk 4]
```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
...
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = ["internal"]
```
Comment (daniel-cit) [inline 1-org/envs/shared/org_policy.tf]: documentation says the value should be `INTERNAL`
https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints#:~:text=The%20list%20of%20allowed%20or%20denied%20values%20can,EXTERNAL

does it work with both `INTERNAL` and `internal` ?

**Review by daniel-cit — CHANGES_REQUESTED** [review]
Looks good.
Some minor changes needed

File: 1-org/envs/shared/org_policy.tf [hunk 5]
```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
...
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/iam.disableServiceAccountKeyUpload"
+}
+
+data "google_organization" "org" {
+  organization = var.org_id
+}
+
+module "domain_restricted_contacts" {
```
Comment (daniel-cit) [inline 1-org/envs/shared/org_policy.tf]: this one should also need the allow_list_length
https://github.com/terraform-google-modules/terraform-google-org-policy/blob/56d3878d0a6a449942143ac8787034f28bb065fb/modules/domain_restricted_sharing/main.tf#L31

File: 1-org/envs/shared/org_policy.tf (hunk 4 above)
Comment (mauro-cit) [reply 1-org/envs/shared/org_policy.tf]: It did not raise any error message so far, but I'll change it to capital anyway.

File: 1-org/envs/shared/org_policy.tf [hunk 6]
```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
...
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = [data.google_organization.org.directory_customer_id]
+  enforce         = "true"
```
Comment (daniel-cit) [inline 1-org/envs/shared/org_policy.tf]: this one should not have  ` enforce = "true"` 
https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/variables.tf#L39-L43

It is not Boolean and it is not deny all / allow all

**Review by daniel-cit — APPROVED** [review]
LGTM

**Pull request closed without merging**